packages/text-document-utils/.gitignore

```diff
@@ -0,0 +1 @@
+
```

Comment:
not sure if we need empty file.

Reply:
removed the empty file, but can be added again if needed in the future.